spec/test_utils.rb

```diff
@@ -0,0 +1,88 @@
+require 'hocon'
+
+module TestUtils
+  Tokens = Hocon::Impl::Tokens
+  EOF = Hocon::Impl::TokenType::EOF
+
+  def TestUtils.wrap_tokens(token_list)
+    # Wraps token_list in START and EOF tokens
+    [Tokens::START] + token_list + [Tokens::EOF]
+  end
+
+  def TestUtils.tokenize_as_list(input_string)
+    origin = Hocon::Impl::SimpleConfigOrigin.new_simple("test")
+    options = Hocon::ConfigParseOptions.defaults
+    io = StringIO.open(input_string)
+
+    token_iterator =  Hocon::Impl::Tokenizer.tokenize(origin, io, options)
```

Comment:
I think it might be good to extract the method up to this point into a `tokenize` function that takes a single input string and returns the `TokenIterator` returned by `Tokenizer.tokenize`. That way, it can be reused later when we port the `tokenizeAsString` method that's going into the Java library.